media-sound/amarok/amarok-9999.ebuild

```diff
@@ -13,13 +13,17 @@ HOMEPAGE="https://amarok.kde.org/"
 
 LICENSE="GPL-2"
 SLOT="5"
-IUSE="ipod lastfm mariadb mtp ofa podcast wikipedia"
+IUSE="cdrom ipod lastfm mariadb mtp ofa podcast wikipedia"
 
-# ipod requires gdk enabled and also gtk compiled in libgpod
+# USE="ipod" requires gdk enabled and also gtk compiled in libgpod.
+# The 'dev-db/mariadb[embedded]' is unavailabe now in portage tree
+# therefore for external MariaDB support the 'dev-db/mariadb-connector-c'
+# is sufficient for DEPEND to build. 'dev-db/mariadb' could be installed later.
 BDEPEND="virtual/pkgconfig"
 DEPEND="
 	>=app-crypt/qca-2.3.0:2
 	>=dev-qt/designer-${QTMIN}:5
+	>=dev-qt/qtcore-${QTMIN}:5
```

Comment:
Unnecessary, as all `ecm.eclass` consumer benefit from the eclass already pulling in this dependency.